rest/src/main/groovy/whelk/rest/api/Crud.groovy

```diff
@@ -943,15 +950,44 @@ class Crud extends HttpServlet {
             failedRequests.labels("DELETE", request.getRequestURI(),
                     HttpServletResponse.SC_BAD_REQUEST.toString()).inc()
             // FIXME data leak
-            response.sendError(HttpServletResponse.SC_BAD_REQUEST,
+            sendError(response, HttpServletResponse.SC_BAD_REQUEST,
                     mve.getMessage())
-        } catch (Exception wre) {
-            log.error("Something went wrong", wre)
+        } catch (Exception e) {
             failedRequests.labels("DELETE", request.getRequestURI(),
                     HttpServletResponse.SC_INTERNAL_SERVER_ERROR.toString()).inc()
-            response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, wre.message)
+            sendError(response, HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e.getMessage(), e)
+        }
+
+    }
+
+    static void sendError(HttpServletResponse response, int statusCode, String msg, Exception e = null) {
+        Map json = [
+                "status_code": statusCode,
+                "status": httpStatus(statusCode),
```

Comment:
Custom method can be replaced by a thing we're already pulling in (did some digging):
```suggestion
                "status": org.eclipse.jetty.http.HttpStatus.getMessage(statusCode),
```